<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgApplyProfile</class>
 <widget class="QWidget" name="WdgApplyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Apply Profile</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>The image data you want to paste does not have an ICM profile associated with it. If you do not select a profile, Krita will assume that the image data is encoded in the import profile defined in the Settings dialog.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblProfile">
          <property name="text">
           <string>&amp;Profiles:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>cmbProfile</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KisSqueezedComboBox" name="cmbProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpRenderIntent">
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>Rendering intent determines the bias in the color conversion.</string>
        </property>
        <property name="title">
         <string>Rendering Intent</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="radioPerceptual">
           <property name="whatsThis">
            <string>Hue hopefully maintained (but not required), lightness and saturation sacrificed to maintain the perceived color. White point changed to result in neutral grays. Intended for images.</string>
           </property>
           <property name="text">
            <string>Perceptual</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioRelativeColorimetric">
           <property name="whatsThis">
            <string>Within and outside gamut; same as Absolute Colorimetric. White point changed to result in neutral grays.</string>
           </property>
           <property name="text">
            <string>Relative colorimetric</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioSaturation">
           <property name="whatsThis">
            <string>Hue and saturation maintained with lightness sacrificed to maintain saturation. White point changed to result in neutral grays. Intended for business graphics (make it colorful charts, graphs, overheads, ...)</string>
           </property>
           <property name="text">
            <string>Saturation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioAbsoluteColorimetric">
           <property name="whatsThis">
            <string>Within the destination device gamut; hue, lightness and saturation are maintained. Outside the gamut; hue and lightness are maintained, saturation is sacrificed. White point for source and destination; unchanged. Intended for spot colors (Pantone, TruMatch, logo colors, ...)</string>
           </property>
           <property name="text">
            <string>Absolute colorimetric</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>KisSqueezedComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbProfile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
